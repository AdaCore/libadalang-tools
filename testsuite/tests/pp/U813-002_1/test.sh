gnatpp --pipe -nM main.adb
echo ""
gnatpp --pipe -nM -cN main.adb
echo ""
gnatpp --pipe -nM -cD main.adb
echo ""
gnatpp --pipe -nM -cU main.adb
echo ""
gnatpp --pipe -nM -cL main.adb
echo ""
gnatpp --pipe -nM -cM main.adb
