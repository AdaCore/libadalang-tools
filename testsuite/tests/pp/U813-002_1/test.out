with Ada.Text_IO; use Ada.Text_IO;
procedure Main is
   Corge_Garply : Integer  := 1;
   Foo_Bar      : constant := 1;
begin
   Put_Line (Foo_Bar'Image);
end Main;

with Ada.Text_IO; use Ada.Text_IO;
procedure Main is
   Corge_Garply : Integer  := 1;
   Foo_Bar      : constant := 1;
begin
   Put_Line (Foo_Bar'Image);
end Main;

with Ada.Text_IO; use Ada.Text_IO;
procedure Main is
   Corge_Garply : Integer  := 1;
   foo_Bar      : constant := 1;
begin
   Put_Line (foo_Bar'Image);
end Main;

with Ada.Text_IO; use Ada.Text_IO;
procedure Main is
   Corge_Garply : Integer  := 1;
   FOO_BAR      : constant := 1;
begin
   Put_Line (FOO_BAR'Image);
end Main;

with Ada.Text_IO; use Ada.Text_IO;
procedure Main is
   Corge_Garply : Integer  := 1;
   foo_bar      : constant := 1;
begin
   Put_Line (foo_bar'Image);
end Main;

with Ada.Text_IO; use Ada.Text_IO;
procedure Main is
   Corge_Garply : Integer  := 1;
   Foo_Bar      : constant := 1;
begin
   Put_Line (Foo_Bar'Image);
end Main;
